CONTRIBUTING.md

```diff
@@ -127,6 +127,245 @@ We try to follow the coding guidelines from the Go community.
 - Code should be commented
 - Code should pass all tests: `make test`
 
+# CLI Guidelines
+This is a design guideline used for development of Flow CLI. Purpose of this guidline
+is to achieve consistency across new features and allow composability of commands that
+build the fundaments of great CLI design.
+
+> Whatever software you’re building, you can be absolutely certain that people will
+> use it in ways you didn’t anticipate. Your software will become a part in a larger
+> system—your only choice is over whether it will be a well-behaved part. - [Clig](https://clig.dev/)
```

Comment:
```suggestion
> Whatever software you’re building, you can be absolutely certain that people will
> use it in ways you didn’t anticipate. Your software will become a part in a larger system
> — your only choice is over whether it will be a well-behaved part. - [Clig](https://clig.dev/)
```